src/routes/index.js

```diff
@@ -1,2 +1,3 @@
 export * from "./home";
 export * from "./login";
+export { Feed } from "./feed";
```

Comment:
Whoops-- this is only needed in `src/components/index.js`, not `src/routes/index.js` 👍